mod/profile/views/default/profile/details.php

```diff
@@ -51,34 +45,37 @@
 				$value = "http://$value";
 			}
 
-			?>
-			<div>
-				<b><?php echo elgg_echo("profile:{$shortname}"); ?>: </b>
-				<?php
-					$params = array(
-						'value' => $value
-					);
-					if (isset($microformats[$shortname])) {
-						$class = $microformats[$shortname];
-					} else {
-						$class = '';
-					}
-					echo "<span class=\"$class\">";
-					echo elgg_view("output/{$valtype}", $params);
-					echo "</span>";
-				?>
+			$class = elgg_extract($shortname, $microformats, '');
+						
+			$field_title = elgg_echo("profile:{$shortname}");
+			$field_value = elgg_format_element('span', [
+				'class' => $class,
+			], elgg_view("output/{$valtype}", [
+				'value' => $value,
+			]));
+			
+			$field = <<<___FIELD
+			<div class='clearfix'>
+				<div class='elgg-col elgg-col-1of5'>
+					<b>{$field_title}:</b>
+				</div>
+				<div class='elgg-col elgg-col-4of5'>
+					{$field_value}
+				</div>
 			</div>
-			<?php
+___FIELD;
 		}
+		
+		$fields_output .= elgg_format_element('div', [
+			'class' => 'profile-fields',
```

Comment:
each field is now wrapped in `profile-fields` this only was one wrapper before. could cause problems